code/modules/mob/mob.dm

```diff
@@ -647,7 +647,14 @@
 	set category = "OOC"
 
 	if (CONFIG_GET(flag/norespawn))
-		return
+		if(!check_rights_for(usr.client, R_ADMIN))
+			to_chat(usr, "<span class='boldnotice'>Respawning in this server is disabled.</span>")
+			return
+		else if("Yes" == alert(src, "Do you want to use your admin privilege to respawn? (Respawning is currently disabled)", "Options", "Yes", "No"))
```

Comment:
No need for the additional log message, and this is cleaner (don't actually commit this directly)

```suggestion
		else if(alert(src, "Do you want to use your admin privilege to respawn? (Respawning is currently disabled)", "Options", "Yes", "No") != "Yes)
		    return
```